.gitignore

```
@@ -14,9 +14,7 @@
 *.ivf
 *.exe
 *.bat
-Build/VS17/x64
 Bin
-Build/linux/debug/
-Build/linux/release/
-
-
+Build
+!Build/linux/build.sh
+!Build/windows/generate_vs17.bat
```

```suggestion
!Build/windows/generate_vs17.bat

```
Please try to add newlines to the eof. That way if someone wants to add something, it doesn't count as a delete and two additions

---

done